tests/test_uncommenter.rb

```diff
@@ -79,6 +79,32 @@
  return (a + b)
 """
 
+SAMPLE_JAVA_COMMENTED = """
```

Comment:
I would say that one test per comment style should be sufficient.

Reply:
I think we should stick with one test per language as the contract is based on language, and the fact that shared code powers this is an internal detail of the class being tested 👍